README.md

```diff
@@ -53,7 +53,7 @@ conduct@dotnetfoundation.org.
 
 ## Officially Released Builds
 
-Download official .NET Core releases [here](https://www.microsoft.com/net/download#core).
+Download official .NET releases [here](https://www.microsoft.com/net/download#core).
```

Comment:
Two lines above we say ".NET Core".

Reply:
It's all a bit confusing during this time of transitioning from .NET Core to just .NET.

The line above concerns licensing, it's specific to the open source .NET so I used .NET Core for now. Maybe it should say ".NET Core and .NET 5 and later"

This line I think should not have Core because the link is for both.

Reply:
May be ".NET Core/.NET"?

Reply:
The page title is "Download .NET" to mean both. I"m not a branding person, but I believe .NET is a general term, with suffixed forms like ".NET Framework 4.8" ".NET Core 3.1" and ".NET 5" to indicate the specific release.